src/main/modules/helmet/modules/contentSecurityPolicy.ts

```diff
@@ -14,7 +14,7 @@ export class ContentSecurityPolicy {
     const additionalClassName = '\'sha256-AaA9Rn5LTFZ5vKyp3xOfFcP4YbyOjvWn2up8IKHVAKk=\''
     const scriptSrc = [inlineJsEnabledBodyClassName, additionalClassName, inlineJsWindowGOVUKClassName, self, '*.google-analytics.com', 'vcc-eu4.8x8.com','vcc-eu4b.8x8.com','www.apply-for-probate.service.gov.uk', 'https://webchat-client.ctsc.hmcts.net']
     const connectSrc = [self, '*.gov.uk', 'https://webchat-client.ctsc.hmcts.net', 'wss://webchat.ctsc.hmcts.net', 'https://webchat.ctsc.hmcts.net', '*.google-analytics.com']
-    const scriptSrcElem = [self, '*.google-analytics.com']
+    const scriptSrcElem = [self, '*.google-analytics.com', 'https://webchat-client.ctsc.hmcts.net', 'wss://webchat.ctsc.hmcts.net', 'https://webchat.ctsc.hmcts.net']
```

Comment:
Aded this to handle the webchat client, this will remove the console log related to CSP violations.